vcpkg/vcpkg.json

```diff
@@ -42,8 +42,9 @@
     "expat",
     {
       "name": "gdal",
-      "default-features": false,
       "features": [
+        "freexl",
+        "kea",
```

Comment:
freexl seems to be shipped in Osgeo4W and debian but not kea. I don't mind having it but what is the proposed policy for vcpkg build? The more driver the better, all drivers can be added ?